calculate/calculate_test.go

```diff
@@ -69,6 +69,7 @@ func TestIncludeExcludePatterns(t *testing.T) {
 	touch(filepath.Join(basePath, "src", "svc.java"))
 	touch(filepath.Join(basePath, "src", "api.js"))
 	touch(filepath.Join(basePath, "src", "nested", "util.js"))
+	touch(filepath.Join(basePath, "src", "nested", "util2.f90"))
```

Comment:
Randomly writing here: lets update the README of this repository that it supports Fortran (I'm referring to https://github.com/apiiro/code-complexity/blob/main/README.md?plain=1#L105)